arai60/01-05_LinkedList/04_082_Remove Duplicates from Sorted List II/level_2.py

```diff
@@ -0,0 +1,21 @@
+# Definition for singly-linked list.
+# class ListNode:
+#     def __init__(self, val=0, next=None):
+#         self.val = val
+#         self.next = next
+class Solution:
+    def deleteDuplicates(self, head: Optional[ListNode]) -> Optional[ListNode]:
+        ans_head = ListNode(0, head)
```

Comment:
```suggestion
        sentinel = ListNode(0, head)
```

sentinel(番兵）とかの方が意図が分かりやすいかも。

Reply:
番兵法という名前だったのですね。
ありがとうございます。